backend/api/tests/views/test_views_function.py

```diff
@@ -36,6 +36,7 @@ class FunctionViewTests(APITestCase):
     client_class = AuthenticatedClient
 
     def setUp(self):
+        self.maxDiff = None
```

Comment:
where is this used? 🤔

Reply:
this is a leftover, it is used in pytest to show the full diff when equlity are. false, good catch!